draft-ietf-quic-manageability.md

```diff
@@ -73,34 +73,33 @@ of QUIC-aware middleboxes, e.g. for load balancing.
 # Introduction
 
 QUIC {{?QUIC-TRANSPORT=I-D.ietf-quic-transport}} is a new transport protocol
-currently under development in the IETF QUIC working group, focusing on
-support of semantics as needed for HTTP/2 {{?QUIC-HTTP=I-D.ietf-quic-http}}.
-Based on current deployment practices, QUIC is encapsulated in UDP and
-encrypted by default. The current version of QUIC integrates TLS
-{{?QUIC-TLS=I-D.ietf-quic-tls}} to encrypt all payload data and most control
-information.
+focusing on support of semantics as needed for HTTP
+{{?QUIC-HTTP=I-D.ietf-quic-http}}.  QUIC is encapsulated in UDP and encrypted
+by default. QUIC integrates TLS {{?QUIC-TLS=I-D.ietf-quic-tls}} to encrypt all
+payload data and most control information. The version of HTTP that operates
+over QUIC is HTTP/3.
```

Comment:
The two mentions of h3 here are disjoint.  Suggest moving the "semantics as needed for HTTP(/3)" later.

Reply:
I would move only the reference to the quic-http draft. Because the first mention if https is the protocol in general, while later it specifically takes about h3.